### DISCUSSION TITLE
Dashboard warning retrieving WebAuthn Credentials.

### DISCUSSION THREAD

**Discussion by KonVei82** [authelia/authelia#6937] [Q&A] [answered]
When Authelia config has:
```
webauthn:
  disable: true
```
then the dashboard will produce the following error:
![image](https://github.com/authelia/authelia/assets/140968638/22f805b7-d6d3-4217-9df7-d0a3608ba0ab)

I would expect the dashboard to read the "disable:true" flag and skip trying to parse any credentials for WebAuthn. 

Is it working as expected right now or is it a glitch?

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

Based on the provided context, it appears that the behavior you're experiencing might not be working as expected. The configuration snippet you've shared explicitly disables WebAuthn by setting `disable: true`. Ideally, this should prevent Authelia from attempting to process or parse any WebAuthn credentials, and thus, you shouldn't encounter errors related to WebAuthn on the dashboard.

However, the error screenshot you've linked (unfortunately, the link is not accessible directly in this format) suggests that despite the `disable: true` configuration, Authelia is still attempting to process WebAuthn credentials, leading to an error. This could be indicative of a glitch or a misconfiguration in how Authelia handles the `disable` flag for WebAuthn.

To address this issue, I recommend the following steps:
1. **Verify Configuration**: Ensure that the configuration file is correctly placed and that Authelia is reading from the correct configuration file. Sometimes, configuration changes might not take effect if Authelia is reading from a different file or if the service hasn't been restarted after making changes.
2. **Review Logs**: Check Authelia's logs for any errors or warnings that might provide more insight into why it's attempting to process WebAuthn credentials despite being disabled.
3. **Update and Restart**: Ensure that Authelia is up to date, as this issue might have been addressed in a more recent version. After updating, restart Authelia to ensure all changes are applied.
4. **Seek Support**: If the issue persists, consider reaching out to the Authelia support channels or filing an issue on the Authelia GitHub repository. Provide them with the configuration snippet, the error screenshot, and any relevant logs to help diagnose the problem.

Without direct access to the error screenshot or more detailed logs, it's challenging to provide a more specific solution. However, the steps above should help in identifying and potentially resolving the issue.

If this doesn't resolve your issue or if you have further questions, please provide additional details or logs for more targeted assistance.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Comment by james-d-elliott** [comment, marked as answer]
That's a bug. Should be relatively easy to fix.

**Comment by HaikoHertes** [comment]
Seems that this issue still / again exists! Have the same issue with latest docker image...

**Comment by nisball** [comment]
I'm experiencing the same on the latest docker image. An OIDC callback broke for one of my services so I looked at the Authelia dashboard & got the same webauthn message. I have it set to disabled.
